### PR TITLE
Enhanced the Login page

### LINKED ISSUE #25: Design a Logo for woodog
### 🐣 Is your feature request related to a problem? Please describe.
Design a new logo for Woodog and place it in the project directory

### LINKED ISSUE #41: UI Enhancements for Login Page
### 🐣 Is your feature request related to a problem? Please describe.

* Enhance the UI of the Login Page. (Refer Figma)

### 📹 Screenshots or Video or GIF

Here's the Figma link,
https://www.figma.com/file/nUaDVjPxi4qn0CTZNoLj0E/Feminine-Divine-Woodog?node-id=0%3A1
(Refer LOGIN Frame)

---

**Note:**
* Do mention if you're working on this issue under ` LGM SOC 2021 `.
* If you want to work on any issue, you should check if it has already been assigned to anyone.

### PR DESCRIPTION
**Pull request by coding-geek21**
## What do these changes do?
Enhanced the Login page

## Related Issue

Fixes #41 

## Checklist:

- [x] Have you checked there aren't other open [Pull Requests](https://github.com/Feminine-Divine/Woodog/pulls) for the same update/change?
- [x] Your submission doesn't break any existing feature.
- [x] Have you tested the code before submission?

## Screenshots or Video or GIF:

https://www.awesomescreenshot.com/video/4131872?key=26747132a48878a68955d6ad12e7f778

---

### CODE REVIEW

File: Woodog/templates/authentication/login.html [hunk 1]
```diff
@@ -1,59 +1,117 @@
...
-  </div>
-  
+        <div class="form-group">
+          <label for="password" class="label">Password</label>
+          <input type="password" class="form-control input-field" name="password" style=" margin-bottom:15px;" id="password">
+          <small class="form-text"><a id="btn-text" href="#">Forgot Password?</a></small>
+        </div>        
+        <div class="form-group">
+            <button type="submit" class="btn btn-login">Login</button>
+            <small class="form-text"><a id="btn-text" href="#">Need an account?</a></small>
```
Comment (hariramjp777) [inline Woodog/templates/authentication/login.html:110]: Please add `href` to this `a`,

```html
<a href="{% url "register" %}" ..>.Need an account?</a>
```

**Review by hariramjp777 — CHANGES_REQUESTED** [review]
Thanks :+1: @coding-geek21, Requested a change,

File: Woodog/templates/authentication/login.html (hunk 1 above)
Comment (coding-geek21) [reply Woodog/templates/authentication/login.html:110]: Sure i will do now @hariramjp777

Comment (coding-geek21) [reply Woodog/templates/authentication/login.html]: Done changes @hariramjp777

File: Woodog/templates/authentication/register.html [hunk 2]
```diff
@@ -118,7 +118,7 @@
       </div>        
       <div class="form-group">
           <button type="submit" class="btn account-btn">Create Account</button>
-          <small class="form-text"><a id="btn-text" href="#">Already have an account?</a></small>
+          <small class="form-text"><a id="btn-text" href="{% url 'login' %}">Already have an account?</a></small>
```
Comment (hariramjp777) [inline Woodog/templates/authentication/register.html:121]: Thanks for doing this too :+1:

File: Woodog/templates/authentication/login.html [hunk 3]
```diff
@@ -107,7 +107,7 @@
         </div>        
         <div class="form-group">
             <button type="submit" class="btn btn-login">Login</button>
-            <small class="form-text"><a id="btn-text" href="#">Need an account?</a></small>
+            <small class="form-text"><a id="btn-text" href="{% url 'register' %}">Need an account?</a></small>
```
Comment (hariramjp777) [inline Woodog/templates/authentication/login.html:110]: :+1:

**Review by hariramjp777 — APPROVED** [review]
**Approved**

**Pull request merged**